ibis/backends/pyspark/__init__.py

```diff
@@ -172,30 +171,35 @@ def version(self):
 
     @property
     def current_database(self) -> str:
-        return self._catalog.currentDatabase()
+        [(db,)] = self._session.sql("SELECT CURRENT_DATABASE()").collect()
```

Comment:
I had to move our catalog operations to use their SQL equivalents because using the pyspark APIs along with the delta catalog results in SQL parse errors. Very odd 🤷🏻